docs/mobile/sdk-advanced-quickstart.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Advanced
+description: Geocortex Web - Learn more about the Geocortex Mobile SDK Quickstart Project
+---
+
+## Development Patterns
+
+For the most part, the Geocortex Mobile project follows the development practices outlined in [Enterprise Application Patterns using Xamarin.Forms](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/enterprise-application-patterns/).
+
+One major deviation from usual Xamarin Development patterns is that [Autofac](https://autofac.org/) is used as the Dependency Injection framework.
+
+### Geocortex Mobile SDK Packages
+
+The Geocortex Mobile SDK is composed of Geocortex packages:
+
+-   `Geocortex Icons`
```

Comment:
These names will need to change at some point. Maybe not part of this PR.

Reply:
Restructuring the Documentation to better organize SDK vs Configuration vs Concepts vs API content in an upcoming ticket, I'll make a note